explorer/testdata/mixin/fail-circular-mixing.carbon

```diff
@@ -13,7 +13,7 @@ __mixin M1 {
   fn F1[me: Self](x: Self) -> Self{
      return x;
   }
-// CHECK: COMPILATION ERROR: {{.*}}/explorer/testdata/mixin/fail-circular-mixing.carbon:[[@LINE+1]]: 'M3' has not been declared yet
+  // CHECK: COMPILATION ERROR: {{.*}}/explorer/testdata/mixin/fail-circular-mixing.carbon:[[@LINE+1]]: 'M3' has not been declared yet
```

Comment:
This change and the ones in the other test files seem unrelated to this PR. Can we revert them?

Reply:
Hm, ofcourse! But I have only run update_checks.py...

Reply:
Yeah, it looks like these files have gotten out of sync with the format that that script wants. I'll send a separate PR to fix them.

Reply:
Done